libs/core/langchain_core/tools/simple.py

```diff
@@ -155,3 +165,6 @@ def from_function(
             args_schema=args_schema,
             **kwargs,
         )
+
+
+Tool.model_rebuild()
```

Comment:
could we add comment about what this does

Reply:
I'll add in all places with a code mod later on -- this is the equvalent of update_forward_refs